README.md

```diff
@@ -638,6 +639,7 @@ When learning CS, there are some useful sites you must know to get always inform
 - [Topic Wise Problem For Competitive Programmer](https://a2oj.com/categories) : Topic wise Practise Problem
 - [Archived Problems - Project Euler](https://projecteuler.net/archives) : Problems Archives
 - [Art of Problem Solving](https://artofproblemsolving.com) : Is math class too easy for you? You've come to the right place!
+- [Canadian Computing Competition](https://cccgrader.com/) : Coding competition held in many Canadian high schools. Presented by the University of Waterloo.  
```

Comment:
We might not need this for now, the pythontutor link is enough. Please can you update accordingly ?

Reply:
OK, removed!